TutorialApplication/RequestManager.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  RequestManager.swift
+//  TutorialApplication
+//
+//  Created by Aleksandr Yegorov on 2/17/17.
+//  Copyright © 2017 Aleksandr Yegorov. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class RequestManager: NSObject
+{
+    static func fetchOrganizations(closure:@escaping ([String : AnyObject])->())
```

Comment:
Code style and argument name:
static func fetchOrganizations(completionHandler: @escaping ([String : AnyObject]) -> ())